spark-operator-api/src/main/java/org/apache/spark/k8s/operator/spec/ExecutorInstanceConfig.java

```diff
@@ -61,8 +62,8 @@
 @Builder
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class InstanceConfig {
-  @Builder.Default protected long initExecutors = 0L;
```

Comment:
`spark.executor.instances` is a `Integer` config.
https://github.com/apache/spark/blob/ae0585226479907e56cef16880849fd03e2bde85/core/src/main/scala/org/apache/spark/internal/config/package.scala#L864-L866